data/secscan_model/secscan_v4_model.py

```diff
@@ -401,8 +401,8 @@ def features_for(report):
                 pkg["version"],
                 [
                     Vulnerability(
-                        vuln["normalized_severity"]
-                        if vuln["normalized_severity"]
+                        vuln["normalized_severity"] if vuln["normalized_severity"] and vuln["normalized_severity"] != PRIORITY_LEVELS["Unknown"]["value"]
+                        else get_priority_from_cvssscore(enrichments[vuln["id"]]["baseScore"]) if enrichments.get(vuln["id"], {}).get("baseScore", None)
                         else PRIORITY_LEVELS["Unknown"]["value"],
```

Comment:
This is getting a little hairy. I guess it makes sense you can chain ternary expressions like this, but it was unexpected.

Reply:
Oh man, and the formatting that black wants to apply makes it even harder to read; I'd suggest pulling this bit of logic out.

Reply:
Sure, I will pull it out.